data/tlg0062/tlg001/tlg0062.tlg001.perseus-grc2.xml

```diff
@@ -52,14 +54,14 @@
 </listBibl>
 </sourceDesc>
 </fileDesc>
- 
+
 <encodingDesc>
 <editorialDecl>
 <correction method="silent">
 <p>optical character recognition</p>
 </correction>
 </editorialDecl>
- 
+
  <refsDecl n="CTS">
   <cRefPattern matchPattern="(\w+).(\w+)"
```

Comment:
the n attributes are missing here. I discovered this issue after this conversion was completed and was preparing a global fix. https://github.com/PerseusDL/canonical-latinLit/issues/515